fitnessllm_dataplatform/services/api_interface.py

```
@@ -0,0 +1,14 @@
+from abc import abstractmethod
+
+
+class APIInterface:
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)